sources/packages/backend/apps/api/src/route-controllers/program-year/program-year.institutions.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { Controller, Get, NotFoundException } from "@nestjs/common";
+import { ProgramYearService } from "../../services";
+import { ClientTypeBaseRoute } from "../../types";
+import BaseController from "../BaseController";
+import { AllowAuthorizedParty } from "../../auth/decorators/authorized-party.decorator";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import { OptionItemAPIOutDTO } from "../models/common.dto";
+import { IsBCPublicInstitution } from "../../auth/decorators";
+
+@AllowAuthorizedParty(AuthorizedParties.institution)
+@Controller("program-year")
+@ApiTags(`${ClientTypeBaseRoute.Institution}-program-year`)
+export class ProgramYearInstitutionsController extends BaseController {
+  constructor(private readonly programYearService: ProgramYearService) {
+    super();
+  }
+
+  /**
+   * Gets a list of program years returned as option items (id/description pair).
+   * @returns an array of program years as id/description objects.
+   */
+  @IsBCPublicInstitution()
+  @Get("options-list")
+  @ApiNotFoundResponse({
+    description: "No program years were found.",
+  })
+  async getProgramYears(): Promise<OptionItemAPIOutDTO[]> {
```

Comment:
I raised the below questions to the business, depending on the answer we can have this centralized in a service controller.
https://github.com/bcgov/SIMS/issues/3062#issuecomment-2093837151